.circleci/config.yml

```diff
@@ -52,207 +61,125 @@ commands:
+      PLATFORM_REGION: "uk-1.platform.sh"
     steps:
+      - hosts_keyscan
       - checkout_code
       - install_php_os_extensions
       - composer_tasks
       - persist_to_workspace:
           root: ./
           paths:
             - ./
-  build_edge_branch:
-    docker:
-      - image: circleci/php:7.4.15-apache-browsers
-    steps:
-      - checkout_code
-      - install_php_os_extensions
-      - run:
-          name: Switch to edge build branch
-          command: |
-            cd ~/project
-            git checkout -b $EDGE_BUILD_BRANCH
-      - run:
-          name: Install dev versions of dof-dss packages with composer.
-          command: |
-            composer require dof-dss/nidirect-site-modules:dev-development dof-dss/nidirect-migrations:dev-development \
-              dof-dss/nidirect-d8-test-install-profile:dev-development dof-dss/nicsdru_origins_theme:dev-development \
-              dof-dss/nicsdru_origins_modules:dev-development dof-dss/nicsdru_nidirect_theme:dev-development
-      - persist_to_workspace:
-          root: ./
-          paths:
-            - ./
-
-  # Test for coding standards.
+  # Test for coding standards - will inherit the workspace/filesystem changes from build step, above.
   coding_standards:
     docker:
       - image: circleci/php:7.4.15-apache-browsers
+    environment:
+      PROJECT_ROOT: "/home/circleci/project"
+      PLATFORM_REGION: "uk-1.platform.sh"
     steps:
       - attach_workspace:
           at: ./
       - run: sudo cp /home/circleci/project/.circleci/docker-php-circleci.ini /usr/local/etc/php/conf.d/
       - run:
           name: PHPCS analysis
-          command: /home/circleci/project/phpcs.sh /home/circleci/project "/home/circleci/project/web/modules/origins /home/circleci/project/web/modules/custom /home/circleci/project/web/modules/migrate /home/circleci/project/web/themes/custom"
+          command: |
+            CHECK_DIRS="${PROJECT_ROOT}/web/modules/custom"
+            CHECK_DIRS="${CHECK_DIRS} ${PROJECT_ROOT}/web/modules/origins"
+            CHECK_DIRS="${CHECK_DIRS} ${PROJECT_ROOT}/web/themes/custom"
+            ~/project/phpcs.sh ${PROJECT_ROOT} $CHECK_DIRS
 
   deprecated_code:
     docker:
       - image: circleci/php:7.4.15-apache-browsers
+    environment:
+      PROJECT_ROOT: "/home/circleci/project"
     steps:
       - attach_workspace:
           at: ./
       - run: sudo cp /home/circleci/project/.circleci/docker-php-circleci.ini /usr/local/etc/php/conf.d/
       - run:
           name: Deprecated code check
           command: |
-            cd /home/circleci/project
-            vendor/bin/drupal-check /home/circleci/project/web/modules/custom /home/circleci/project/web/modules/origins /home/circleci/project/web/modules/migrate /home/circleci/project/web/themes/custom -e "*/tests/src/*"
-
-  # Run any unit tests and any kernel tests against a vanilla D8 site + our site config imported over it (no predefined content).
-  unit_kernel_tests:
```

Comment:
Removed (for now) as we can run these as part of our edge build, or in a dedicated nightly workflow